docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - 8000:8000
+    volumes:
+      - ./:/app
+    command: >
+      bash -c " while !</dev/tcp/db/${DJANGO_DATABASE_PORT}; do sleep 1; done;
+                python manage.py migrate &&
+                python manage.py runserver 0.0.0.0:8000"
```

Comment:
Move out of here the migrate command, it is unlikely that anyone would want to run migrations every time the container starts. Put it in the readme file as required step for initial setup.

Reply:
done